src/transport_ofi.h

```diff
@@ -233,11 +233,16 @@ typedef struct shmem_transport_ofi_bounce_buffer_t shmem_transport_ofi_bounce_bu
 
 typedef int shmem_transport_ct_t;
 
-#ifdef __APPLE__
-#define TID_TYPE uint64_t
-#else
-#define TID_TYPE pid_t
-#endif
+enum shmem_internal_tid_t { PID_T, UINT64_T };
```

Comment:
Seems possible these names could clash since they differ from the types only in case.  What about something like `tid_is_pid_t` and `tid_is_uint64_t`?